sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -20,13 +20,13 @@ Install the Azure Form Recognizer client library for .NET with [NuGet][nuget]:
 dotnet add package Azure.AI.FormRecognizer
 ``` 
 
-> Note: This version of the client library defaults to the `v2.1-preview.3` version of the service.
+> Note: This version of the client library defaults to the `v2.1` version of the service.
 
 This table shows the relationship between SDK versions and supported API versions of the service:
 
 |SDK version|Supported API version of service
 |-|-
-|3.0.0 - Latest GA release | 2.0
+|3.0.0 | 2.0
```

Comment:
should this include 3.1.0 ?

Reply:
yeah.. I actually added it and then delete it... idk I am torn. So if I add it now it will make it to code complete and yay!
but I wonder if it might confuse users that look at our Github readme between now and when we actually release.... :(
What do u think?

Reply:
I side with updating it now. You could be `(Unreleased)` next to it to be super clear

Reply:
will do